### DISCUSSION TITLE
point classé

### DISCUSSION THREAD

**Discussion by Voisin1234** [Crownicles/Crownicles#2380] [Nouvelles idées]
Bonjour à tous, je souhaitez-vous faire partager une idée qui ne va pas plaire à tout le monde je le sais. L'idée est très simple, c'est que tous les combats qui se font sur l'ile et tous les combats classés soient possible de se faire uniquement sur le discord draftbot . Cela éviterait qu'il y ait qui feed entre eux sur d'autres discord. Pour l'île , ça éviterait certains de chercher des bugs absolument et qui ne sont pas forcément vérifiable par le staff .Pour les combats sur l'ile si les personnes veulent que ce soit caché car si ils ont des techniques ou autre et qu'ils ne veulent pas que les autres joueurs, le voient il pourrait y avoir des Chanel privé ou seul les administrateurs pourraient y accéder. Sauf si certains veulent jouer en public. Je pense que cela déjà pourrait régler quelques problèmes d'accusation. J'attends vos messages de contradiction.😙

**Comment by romain22222** [comment]
je suis contre perso : l'une des forces de draftbot c'est que ce soit "multi", si on enlève un gros morceau hors serv principal on perd de cette force. Je serais plus pour attendre le matchmaking, qui par son principe même permettrait une connexion avec les autres plus facilement

**Comment by 0eddo** [comment]
Hello, effectivement un message de contradiction 😅 
Je pense que l'idée est assez mauvaise tt simplement parce que le fait de tt centraliser sur le serv offi ça va faire perdre des joueurs qui jouent occasionnellement pour le divertissement. Les missions de combats sont déjà assez complexe pour eux et l'île aussi pour qu'en plus ils doivent rejoindre un autre serveur alors que ce sont des joueurs "plaisir" principalement. En + le feed bah honnêtement ce ne pourra jamais être totalement restreint yaura tt le temps des gens qui trouveront une façon ou une autre de feed que ce soit sur un autre serv ou sur l'offi. En plus ça casserait un peu l'effet communauté notamment sur les servs externes de guildes si on restreint un des seuls moyens d'intéractivité du jeu.

**Reply by Voisin1234** [reply]
Alors je ne parle pas forcément des petits joueurs qui jouent occasionnellement, sachant que les trois quarts les personnes qui sont dans les premières pages, que ce soit dans le classement général ou dans le classement, glorieux sont tous déjà sur le discorde officiel. Et, en toute honnêteté, les gens qui jouent occasionnellement ne m’importe peu. Après pour ce qui est des missions de combat, je ne vois pas ce que ça peut changer car ils peuvent très bien les faire en amicaux. Dans mes souvenirs il y a aucune mission de combat classé. De plus si ils veulent trouver, vraiment trouver des un maximum adversaires le mieux du coup c’est le discorde officiel.  Pour l’île si c’est des joueurs qui sont tout seul, effectivement ils peuvent le faire n’importe où ça va pas changer grand-chose pour nous pour le classement. Pour laisser communautaire, rien de t’empêche de faire tout le reste, la vente d’animaux etc. etc. Mais je comprends totalement ton point de vue, personnellement je ne trouve pas que ce soit des bons arguments

**Comment by 0eddo** [comment]
Il y a pourtant les missions avoir 100/800/1200 de gloire 🙃

**Reply by Voisin1234** [reply]
Bah justement raison de plus pour être sur le discorde officiel pour trouver un maximum d’adversaire car si t’es sur un discord ou vous êtes deux, ça va être compliqué de monter très vite

**Comment by 0eddo** [comment]
Oui, oui mais ce que je dis justement c que si ts les combats sont concentrés sur le serv offi, ça peut justement déranger et décourager les joueurs occasionnels 🙂 j'en connais pas mal notamment ceux avec qui j'avais commencé le jeu qui ne veulent/voulaient pas rejoindre le serveur offi car ils voulaient jouer trkl ds leur coin entre eux :). Le prblm c'est que si on les contraint à faire des actions que ce soit pve etc sur l'offi ça décourage. Bien évidemment ils représentent peut être une petite partie des joueurs mais il faut compter avec :)

**Comment by Voisin1234** [comment]
Je suis déçu, je m'attendais à plus de contradiction 😅

**Comment by BastLast** [comment]
Je vais être honnête mon plan actuel c'est de supprimer les. Combats classés du jeu et de les remplacer par autre chose

**Reply by Voisin1234** [reply]
Peux-tu nous en dire un peu plus ?

**Reply by BastLast** [reply]
Nope
